std/experimental/allocator/building_blocks/free_list.d

```diff
@@ -1045,6 +1047,28 @@ struct SharedFreeList(ParentAllocator,
         resetNodes();
         return result;
     }
+
+    /**
+    Nonstandard function that minimizes the memory usage of the freelist by
+    freeing each element in turn. Defined only if $(D ParentAllocator) defines
+    $(D deallocate).
+    */
+    static if (hasMember!(ParentAllocator, "deallocate") && !unchecked)
```

Comment:
The description and constraint were copied from FreeList.minimize().  The unchecked constraint looked unnecessary to me, but I am not running into issues with the limitation, so I played it conservatively and left it alone.  Perhaps it was placed intentionally for an important reason.  If you wrote FreeList.minimize(), can you remember why the constraint is there?  If we are confident, then I think it makes sense to either document the rationale or remove it from both places, and I'd be perfectly willing to do that.

Now that I look at it in more detail, I wonder if "unchecked" should be defined in the positive instead and called "bounded".  It is read 5 times in FreeList, 1 of which it appears as "unchecked" and 4 of which it appears as "!unchecked".  I remember that when I first saw it, it was a little confusing because I couldn't guess what it was checking.